wrapper/tests/test_pre_copy.py

```diff
@@ -104,3 +105,63 @@ def test_uri_parsing_full(self):
         self.assertEqual(vmw.user, 'user@domain')
         self.assertEqual(vmw.server, 'some.remote.server')
         self.assertEqual(vmw.port, 443)
+
+    def test_get_index_string_basic(self):
+        """ Test mapping disk indices to string. """
+        self.assertEqual(_get_index_string(0), 'a')
+        self.assertEqual(_get_index_string(1), 'b')
+        self.assertEqual(_get_index_string(5), 'f')
+        self.assertEqual(_get_index_string(25), 'z')
+
+        # Last positions in multi-digit numbers is treated differently, because
+        # that's how the naming works out.
+
+        #                                  a      a
+        self.assertEqual(_get_index_string(1 * 26 + 0), 'aa')
+        #                                  a             a        a
+        self.assertEqual(_get_index_string(1 * 26 * 26 + 1 * 26 + 0), 'aaa')
+
+        self.assertEqual(_get_index_string(1 * (26 ** 3) +  # a
+                                           1 * (26 ** 2) +  # a
+                                           1 * (26 ** 1) +  # a
+                                           0 * (26 ** 0)),  # a
+                         'aaaa')
+
+        self.assertEqual(_get_index_string(1*(26**4) +  # you
+                                           1 * (26 ** 3) +  # get
+                                           1*(26**2) +  # the
+                                           1*(26**1) +  # point,
+                                           0*(26**0)),  # right?
+                         'aaaaa')
```

Comment:
`you get the point, right?` ... well, not quite yet. Please add a test that contains something besides "a". How do you "spell" `abc` for example?

Ok, I can see that in the test below, but you still could add at least one here.